polar_route/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.2.16"
+__version__ = "0.2.17"
```

Comment:
The 0.2.x branch is already at 0.2.17 so this will need to be updated when the PR is ready to merge